rel/files/couchdb.in

```diff
@@ -31,8 +31,10 @@ export COUCHDB_QUERY_SERVER_COFFEESCRIPT="{{prefix}}/bin/couchjs {{prefix}}/shar
 export COUCHDB_FAUXTON_DOCROOT={{fauxton_root}}
 
 ARGS_FILE="${COUCHDB_ARGS_FILE:-$ROOTDIR/etc/vm.args}"
+INI_FILES=${COUCHDB_INI_FILES:-}
 SYSCONFIG_FILE="${COUCHDB_SYSCONFIG_FILE:-$ROOTDIR/releases/$APP_VSN/sys.config}"
 
 exec "$BINDIR/erlexec" -boot "$ROOTDIR/releases/$APP_VSN/couchdb" \
      -args_file "${ARGS_FILE}" \
+     -couch_ini ${INI_FILES} \
```

Comment:
No `"` here, because `$INI_FILES` can reference zero or more files, and they must be handled as such.